components/WelcomeBanner.vue

```diff
@@ -3,9 +3,9 @@
         <div class="flex-row banner-container">
             <div class="flex-row banner-content column">
                 <h1 class="banner-header">
-                    <span>Discover and participate in Scratch game jams</span>
+                    <span>Discover and Participate in Scratch Game Jams</span>
                     <br />
-                    <span>Embrace the community</span>
+                    <span>Embrace the Community</span>
```

Comment:
I think that these titles would look better uncapitalized. On Scratch, there's a similar banner where such titles are not capitalized.

Reply:
ok